pkg/juicefs/juicefs.go

```diff
@@ -204,9 +203,11 @@ func (j *juicefs) MountFs(volumeID, name string, options []string) (string, erro
 		return mountPath, status.Errorf(codes.Internal, "Could not check mount point %q exists: %v", mountPath, err)
 	}
 
+	args := append([]string{"mount", name, mountPath}, options...)
```

Comment:
`j.Mount` will add `-o` before `options`, so `"-o"` need to be in `args`

Reply:
`args := append([]string{"mount", name, mountPath, "-o"}, options...)` @yujunz 

`options` are like `cache-dir=/var/foo,cache-size=100`

Reply:
OK. Will correct and test it before merging